src/coreclr/scripts/exploratory.proj

```diff
@@ -0,0 +1,87 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+
+  <!-- <PropertyGroup>
```

Comment:
You're keeping this, for future ease of debugging? If so, maybe add a comment to that effect. (and how to use it)

Reply:
Deleted it as similar comment is present in other proj files.